forc/src/ops/forc_init.rs

```diff
@@ -4,6 +4,8 @@ use std::fs;
 use sway_utils::constants;
 
 pub(crate) fn init_new_project(project_name: String) -> Result<()> {
+    let neat_name:String = project_name.split("/").last().unwrap().to_string();
+
     // Make a new directory for the project
     fs::create_dir_all(format!("{}/src", project_name))?;
```

Comment:
Ahh unrelated to this PR, but we shouldn't be using `format!` to create paths like this as it breaks Windows support. I'll open a separate issue for this so we can address it in a future PR.

Reply:
Yeah I was contemplating using Path from the Rust standard library but that seems like another refactor in itself

Reply:
I've opened #872